### PR TITLE
Swap all repeats to multiple='True' where appropriate

### PR DESCRIPTION
**Pull request by erasche**
In support of https://github.com/galaxyproject/tools-iuc/issues/239

- [x] Depends on galaxyproject/galaxy#749

### CODE REVIEW

File: tools/piranha/piranha.xml [hunk 1]
```diff
@@ -46,16 +46,14 @@
...
         #end for
 
         -o ./piranha.out
     ]]></command>
     <inputs>
         <param name="input" type="data" format="bed,bam" label="Choose input BED or BAM file"/>
-        <repeat name="covariates" title="Choose covariate">
-            <param name="covariate_file" type="data" format="bed" label="Choose covariate BED file"/>
-        </repeat>
+        <param name="covariates" type="data" format="bed" label="Choose covariate BED file"/>
```
Comment (bgruening) [inline tools/piranha/piranha.xml]: multiple missing

File: tools/homer/tools/makeTagDirectory.xml [hunk 2]
```diff
@@ -25,9 +25,7 @@
     <inputs>
         <expand macro="input_choose_homer_version" />
 
-        <repeat name="alignment_files" title="Alignment Files">
-            <param name="file" label="Add file" type="data" format="sam,bed,bam" help="Alignments in SAM, BAM or BED format" />
-        </repeat>
+        <param name="alignment_files" label="Alignment Files" type="data" format="sam,bed,bam" help="Alignments in SAM, BAM or BED format" />
```
Comment (bgruening) [inline tools/homer/tools/makeTagDirectory.xml]: multiple=True is still missing

File: tools/piranha/piranha.xml [hunk 3]
```diff
@@ -115,12 +113,7 @@
     <tests>
         <test>
             <param name="input" value="zntbr_unbinned_response.bed" ftype="bed"/>
-            <repeat name="covariates">
-                <param name="covariate_file" value="ztnbr_binning_cov1.bed" ftype="bed"/>
-            </repeat>
-            <repeat name="covariates">
-                <param name="covariate_file" value="ztnbr_binning_cov2.bed" ftype="bed"/>
-            </repeat>
+            <param name="covariates_file" value="ztnbr_binning_cov1.bed,ztnbr_binning_cov2.bed" ftype="bed"/>
```
Comment (bgruening) [inline tools/piranha/piranha.xml]: It should be named `covariates`.

File: tools/homer/tools/makeTagDirectory.xml (hunk 2 above)
Comment (erasche) [inline tools/homer/tools/makeTagDirectory.xml]: Ugh...so sorry!

**Comment by bgruening** [comment]
Awesome! max="" and min="" is also now available - I will merge.

**Pull request merged**